draft-ietf-teep-protocol.xml

```diff
@@ -146,11 +140,10 @@
 
     <section anchor="messages" title="Message Overview">
       <t>The TEEP protocol consists of a couple of messages exchanged between a TAM and a TEEP Agent via a TEEP Broker. 
-      The messages are encoded either in JSON or CBOR and designed to provide end-to-end security. 
-      TEEP protocol messages are signed and/or encrypted by the endpoints, i.e., the TAM and the TEEP Agent, but trusted 
+      The messages are encoded in CBOR and designed to provide end-to-end security. 
+      TEEP protocol messages are signed by the endpoints, i.e., the TAM and the TEEP Agent, but trusted 
       applications may as well be encrypted and signed by the service provider. The TEEP protocol not only re-use 
-      JSON and CBOR but also the respective security wrappers, namely JOSE (JWS <xref target="RFC7515"/>
-          and JWE <xref target="RFC7516"/>, to be more specific) and COSE <xref target="RFC8152"/>. Furthermore, 
+      CBOR but also the respective security wrapper, namely COSE <xref target="RFC8152"/>. Furthermore, 
```

Comment:
the word "respective" doesn't belong when there's only one alternative.  Remove it.